src/sentry/search/events/datasets/discover.py

```diff
@@ -1020,6 +1024,18 @@ def function_converter(self) -> Mapping[str, SnQLFunction]:
                     ),
                     private=True,
                 ),
+                SnQLFunction(
+                    "salted_column_hash",
+                    # TODO: figure out how to make this take arbitrary number of arbirary type arguemnts
```

Comment:
This is not something we support in the query builder today. Is there a reason why you want this?

Reply:
It's jut that `farmFingerprint64` (and `cityHash64` for that matter) take arbitrary number of arbitrary params. Which as you said is not supported by the QueryBuilder. Not something needed for this particular PR, but would give more flexibility moving forward.